src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/SuggestedAction.cs

```diff
@@ -308,5 +312,18 @@ public override int GetHashCode()
         }
 
         #endregion
+
+        internal TestAccessor GetTestAccessor()
+            => new TestAccessor(this);
+
+        internal readonly struct TestAccessor
+        {
+            private readonly SuggestedAction _suggestedAction;
+
+            public TestAccessor(SuggestedAction suggestedAction)
+                => _suggestedAction = suggestedAction;
+
+            public ref bool IsApplied => ref _suggestedAction._isApplied;
```

Comment:
is this acutally set in tests?

Reply:
no one seems to assign.  i would just expose as bool.

Reply:
This form means the tests operate as though they directly have access to the backing field. The backing field is mutable.

Reply:
I'm on hte fence.  I get the desire here.  But i also think it's valuable if hte test accessor (or, frankly, any consumer) can have minimal access.  But if you feel this is really a core part of the design, i'm fine with you preserving this.